src/pages/VideoPage/ModalAddToPlaylist.jsx

```diff
@@ -0,0 +1,155 @@
+import { useState } from 'react';
+import faker from 'faker';
+import { IoIosCloseCircle } from 'react-icons/io';
+import { MdPlaylistAdd } from 'react-icons/md';
+import { BtnIcon, Btn } from '../../components/morphine-ui';
+import { useLibraryContext } from '../../context/libraryState';
+import { AddToPlaylistBtn } from './AddToPlaylistBtn';
+
+export const ModalAddToPlaylist = ({
+  desiredVideoId,
+  showModal,
+  setShowModal,
+}) => {
+  const {
+    state: { playlists, videos },
+    dispatch: playlistDispatch,
+  } = useLibraryContext();
+  const [newPlaylistName, setNewPlaylistName] = useState('');
+
+  return (
+    <dvi
+      className="flex flex--column align-items--c justify-content--c"
+      style={
```

Comment:
remove style as all the content is commented

Reply:
done